reactDashboard/src/components/GeoHeatMap/FullscreenControl.css

```diff
@@ -1,5 +1,30 @@
 .fullscreen svg {
-    height: 25px;
-    width: 25px;
-    color: #319bd5;
+  height: 25px;
+  width: 25px;
+  color: #133a53;
+}
+
+.fullscreen:hover,
+.fullscreen svg:hover {
+  color: white;
+  background-color: #133a53;
+}
+
+.darkmode .fullscreen:hover,
+.darkmode .fullscreen svg:hover {
+  color: #c1def1;
+  background-color: #121212;
+}
+
+.darkmode .fullscreen:hover {
+  border: 1px solid #c1def1;
```

Comment:
Why does only darkmode have a border?

Reply:
✅

Reply:
border is added to separate the button background from the map background

Reply:
Could we have a border all the time and just override the darkmode border colour?

Reply:
refactored